devtools/parquet_to_duckdb.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+"""Script that creates a DuckDB database from a collection of PUDL Parquet files."""
+
+import logging
+from pathlib import Path
+
+import click
+import sqlalchemy as sa
+
+from pudl.metadata import PUDL_PACKAGE
+from pudl.metadata.classes import Package
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("parquet_dir", type=click.Path(exists=True, resolve_path=True))
+@click.argument(
+    "duckdb_path", type=click.Path(resolve_path=True, writable=True, allow_dash=False)
+)
+def convert_parquet_to_duckdb(parquet_dir: str, duckdb_path: str):
+    """Convert a directory of Parquet files to a DuckDB database.
+
+    Args:
+        parquet_dir: Path to a directory of parquet files.
+        duckdb_path: Path to the new DuckDB database file (should not exist).
+
+    Example:
+        python parquet_to_duckdb.py /path/to/parquet/directory duckdb.db
+    """
```

Comment:
Further down the road do we want this functionality to stay outside of the core package? Or do we think we'll want to run it at the end of the ETL all the time? Would it make sense to put it in say `pudl.convert.parquet_to_duckdb`? and add the CLI to our entry points in `pyproject.toml` so we can test it along with all the other CLIs?

Reply:
Yeah let's add it to `pudl.convert` so we can test it with our other CLIs.